playbooks/PnP.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Manage operations add device, claim device and unclaim device of Onboarding Configuration(PnP)
```

Comment:
- name: Manage operations - Add, claim, and unclaim devices of Onboarding Configuration (PnP)